README.md

```diff
@@ -1 +1,49 @@
-# My-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Desktop version Personal Portfolio and deploy to online
```

Comment:
[OPTIONAL] Kindly use a short descriptive name for your Project e.g. `Portfolio` or `My-Portfolio` 🤗